core/devdoc/outprocess_loader_requirements.md

```diff
@@ -34,6 +34,8 @@ typedef struct OUTPROCESS_LOADER_ENTRYPOINT_TAG
     STRING_HANDLE control_id;
     /** @brief The URI for the gateway message channel.*/
     STRING_HANDLE message_id;
+    /** @brief controls timeout for ipc retries. */
+    int default_wait;
```

Comment:
nanomsg is actually expecting an int, but for me saving it, it should be unsigned int.